PolyDeploy/Clients/Manage/src/js/templates/users.html

```diff
@@ -11,15 +11,21 @@ <h3 class="panel-title">New API User</h3>
                 </div>
                 <div class="panel-body">
 
-                    <div class="form-group">
+                    <div class="form-group row">
```

Comment:
`row` isn't actually needed here. However removing it made me realise that the cause of the layout issue is that I forgot to wrap all the `form-group` divs with a `form-horizontal` div.

http://getbootstrap.com/docs/3.3/css/#forms-horizontal

I think the columns also need a little tweak, 3 for the title and 9 for the input should tidy it up.

I can make these tweaks if preferred 🙂.

Reply:
:+1: go for it